### PR TITLE
Display balances in category autocomplete

### PR DESCRIPTION
**Pull request by joel-jeremy**
Closes https://github.com/actualbudget/actual/issues/534

### CODE REVIEW

**Comment by rich-howell** [comment]
This is exactly what I wanted - only thing I would say is the red is really hard to read.

**Comment by Teprifer** [comment]
I concur, in dark and midnight themes the colours are fine, but in light even the green doesn't have good readability - especially if highlighted:

image

**Comment by joel-jeremy** [comment]
I'm frying to figure out how to make the balances available to the test which is failing. Would appreciate any tips to put me into the right direction. :)

**Comment by Teprifer** [comment]
Desktop contrasts are much improved, thanks :)
For the midnight theme I think the menu highlight looks too bright (to me anyway).

image

On mobile, light mode, category names can't be seen until highlighted:

image

**Comment by joel-jeremy** [comment]
Did a rebase, the mobile above should now be resolved. :)

**Comment by MatissJanis** [comment]
Small bug on desktop version:

1. select 2x transactions
2. click "2 transactions" to open the menu and then `category`
3. fatal crash

**Comment by joel-jeremy** [comment]
The batch edit bug should now be fixed :)

File: packages/desktop-client/src/components/budget/util.ts [hunk 1]
```diff
@@ -67,14 +67,20 @@ export function makeAmountStyle(
...
+  value: number,
+  colors?: {
+    positiveColor?: string;
+    negativeColor?: string;
+    zeroColor?: string;
+  },
+) {
+  const _positiveColor = colors.positiveColor || theme.noticeText;
+  const _negativeColor = colors.negativeColor || theme.errorText;
+  const _zeroColor = colors.zeroColor || theme.tableTextSubdued;
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/budget/util.ts:80]: **🥜 nitpick:** ‏tbh I'm not a big fan of this naming convention. I don't see it adding any value. Especially in this context.

**Review by MatissJanis — APPROVED** [review]

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**